mteb/tasks/Clustering/eng/BiorxivClusteringS2S.py

```diff
@@ -1,10 +1,50 @@
 from __future__ import annotations
 
 from mteb.abstasks.AbsTaskClustering import AbsTaskClustering
+from mteb.abstasks.AbsTaskClusteringFast import AbsTaskClusteringFast
 from mteb.abstasks.TaskMetadata import TaskMetadata
 
 
+class BiorxivClusteringS2SFast(AbsTaskClusteringFast):
+    metadata = TaskMetadata(
+        name="BiorxivClusteringS2SFast",
+        description="Clustering of titles from biorxiv across 26 categories.",
+        reference="https://api.biorxiv.org/",
+        dataset={
+            "path": "mteb/biorxiv-clustering-s2s",
+            "revision": "756e4c908ca0888e048aaa0b5ffbf3ea163cf916",
+        },
+        type="Clustering",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="v_measure",
+        date=("2021-01-01", "2022-05-10"),
+        form=["written"],
+        domains=["Academic"],
+        task_subtypes=["Thematic clustering"],
+        license="https://www.biorxiv.org/content/about-biorxiv",
```

Comment:
Please check the license what it is?
Examples:

license="MIT"
license="CC-BY-NC-SA-4.0"
license="Not specified"
license=None

Reply:
I think the link is good if it is not a standard license. @Muennighoff did you do anything do select specific licenses during construction?

Reply:
We had a convo about this jesterday with Jan

Reply:
I also think it is fine for a merge (just wanted to know if the original datasets filtered on author selected licenses)

Reply:
it looks like licenses weren't selected, here's the api call
https://github.com/embeddings-benchmark/mteb/blob/main/scripts/data/biorxiv/script_raw

i'm gonna keep the link then if that's the consensus

Reply:
Looks fine to me! Also cc @loicmagne